RELEASE_NOTES.md

```diff
@@ -1,5 +1,32 @@
 # Release Notes
 
+## 4.5.1
+### Security
+- Do not include image annotations when building spec. These annotations can have security implications - crun, for example, allows rootless containers to preserve the user's groups through an annotation.
+
+### Changes
+- The image registry has been migrated to registry.k8s.io
```

Comment:
I don't know what that means and I think it is really confusing for users. We do not have a default registry.

Reply:
yes, i'll remove that one.

Reply:
done